unit_test/sample_key/readme.txt

```diff
@@ -618,3 +618,52 @@ openssl asn1parse -in end_responder_with_spdm_req_rsp_eku.cert -out end_responde
 openssl asn1parse -in end_responder_with_spdm_req_eku.cert -out end_responder_with_spdm_req_eku.cert.der
 openssl asn1parse -in end_responder_with_spdm_rsp_eku.cert -out end_responder_with_spdm_rsp_eku.cert.der
 popd
+
+==== More alias_cert model cert_chain to gen ====
+NOTE: The bundle_responder.certchain_alias.der and bundle_requester.certchain.der have same ca_cert and inter cert.

```

Comment:
Please add:
This alias cert chain is partial, from root CA to device certificate CA.

Reply:
Thanks. I have added the comment.